backend/azure-functions/HttpTrigger/function.json

```diff
@@ -12,5 +12,6 @@
       "direction": "out",
       "name": "res"
     }
-  ]
+  ],
+  "scriptFile": "../dist/HttpTrigger/index.js"
```

Comment:
this is a bit problematic, we'll need it at the build folder eventually because it is the one we are uploading to azure